README.md

```diff
@@ -12,8 +12,7 @@ This is a collection of IPython notebooks that I use to teach topics relating to
 - [Map-Reduce programming and Apache Spark](http://nbviewer.jupyter.org/github/phelps-sg/python-bigdata/blob/master/src/main/ipynb/spark-mapreduce.ipynb)
 - [Column-oriented databases with HBase and HappyBase](http://nbviewer.jupyter.org/github/phelps-sg/python-bigdata/blob/master/src/main/ipynb/hbase-python-slides.ipynb)
 
-A virtual-machine containing all of the prerequisite software can be downloaded [here](https://www.dropbox.com/s/nkv9s765y9peg0h/datascience-lm.ova?dl=0).
-This can be imported as a virtual appliance using [VirtualBox](https://www.virtualbox.org/).
+A Virtual Appliance can be imported using [VirtualBox](https://www.virtualbox.org/).
```

Comment:
Thanks for the contribution.  Can you remove the sentence "A Virtual Applicance can be imported.." as this is now redundant.

Reply:
Okay.

Reply:
I have done what you asked.